### PR TITLE
Some fonts have spaces in the name

### PR DESCRIPTION
**Pull request by bitboxer**
To be able to copy files with spaces I added some quotes to the cp command :wink:

I could also use Shellescape to do this if you prefer.

### CODE REVIEW

File: spec/unit/fonts_spec.rb [hunk 1]
```diff
@@ -24,9 +24,9 @@
 
     it 'should copy fonts if they are not found' do
       expect(executor).to receive(:execute)
-        .with('Copying roboto.ttf', "cp #{config.fonts_path}/roboto.ttf #{ENV['HOME']}/Library/Fonts/roboto.ttf")
+        .with('Copying roboto.ttf', "cp '#{config.fonts_path}/roboto.ttf' '#{ENV['HOME']}/Library/Fonts/roboto.ttf'")
```
Comment (bitboxer) [inline spec/unit/fonts_spec.rb]: Hound, sometimes you are really naiv.

**Comment by bitboxer** [comment]
I will rebase this to master as soon as the bundler pr is done, than this should be green again

**Comment by moonglum** [comment]
Rebase, please 😁

**Comment by bitboxer** [comment]
This is green now :+1:

**Comment by moonglum** [comment]
Great! 
(A little confused by Hound's confusion oO How do they do the parsing?! This is... sad oO)

**Comment by moonglum** [comment]
Thank you, Mr. @bitboxer :heart:

**Comment by bitboxer** [comment]
Regexp all the way!!

**Pull request merged**